### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
Updated main MCS README to link to other public MCS GitHub repositories. (Other minor cleanup too, including adding a table of content.)

### CODE REVIEW

File: machine_common_sense/DEV.md [hunk 1]
```diff
@@ -171,10 +177,3 @@ Desired screen width. If value given, it must be more than `450`. If none given,
 (string)
 
 Team name identifier to prefix to filenames uploaded to S3 (default: '').
-
-## References
-
-- AI2-THOR Documentation: http://ai2thor.allenai.org/documentation
-- AI2-THOR GitHub: https://github.com/allenai/ai2thor
-- MCS AI2-THOR GitHub Fork: https://github.com/NextCenturyCorporation/ai2thor
-- MCS AI2-THOR Scene Files and Schema: [scenes](./machine_common_sense/scenes)
```
Comment (ThomasSchellenbergNextCentury) [inline machine_common_sense/DEV.md:180]: Removed this since it's all in the main README now

File: README.md [hunk 2]
```diff
@@ -298,14 +311,28 @@ From your python environment, run test.py and check the output images for proper
...
 ## Troubleshooting
 
 [mcs-ta2@machinecommonsense.com](mailto:mcs-ta2@machinecommonsense.com)
 
 ## Apache 2 Open Source License
 
-Code in this repository is made available by [Next Century
-Corporation][1] under the [Apache 2 Open Source License][2].  You may
+Code in this repository is made available by [CACI][4] (formerly [Next Century
+Corporation][1]) under the [Apache 2 Open Source License][2].  You may
```
Comment (rartiss55) [inline README.md:335]: Fun fact: the old Next Century link now redirects to CACI.

File: README.md [hunk 3]
```diff
@@ -298,14 +311,28 @@ From your python environment, run test.py and check the output images for proper
...
+- [Python API](./machine_common_sense/API.md)
+- [Example Scene Configuration Files](./machine_common_sense/scenes/README.md)
+- [Scene Configuration JSON Schema](./machine_common_sense/scenes/SCHEMA.md)
+- [Developer Docs](./machine_common_sense/DEV.md)
+
+## Other MCS GitHub Repositories
+
+- [Unity code](https://github.com/NextCenturyCorporation/ai2thor)
+- [Scene Generator](https://github.com/NextCenturyCorporation/mcs-scene-generator)
+- [Data Ingest](https://github.com/NextCenturyCorporation/mcs-ingest)
```
Comment (rartiss55) [inline README.md:325]: Don't know if we need the UI/ingest links here but I suppose it doesn't hurt.

**Review by rartiss55 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ThomasSchellenbergNextCentury) [reply README.md:325]: I figured we should be as transparent with our repos as possible.

**Pull request merged**